Clarified the traces_sample_rate behavior in Python. (#8561)
getsentry/sentry-docs: src/platforms/common/configuration/options.mdx

A number between 0 and 1, controlling the percentage chance a given transaction will be sent to Sentry. (0 represents 0% while 1 represents 100%.) Applies equally to all transactions created in the app. Either this or <PlatformIdentifier name="traces-sampler" /> must be defined to enable tracing.

<PlatformSection supported={["python"]}>

> If <PlatformIdentifier name="traces-sample-rate" /> is 0 this means that no new traces will be created. But if you have another service (for example an JS frontend) that makes requests to your service that include trace information, that traces will be continued and thus transactions will be sent to Sentry.

Reviewer: ```suggestion
If <PlatformIdentifier name="traces-sample-rate" /> is 0, this means that no new traces will be created. However, if you have another service (for example a JS frontend) that makes requests to your service that include trace information, those traces will be continued and thus transactions will be sent to Sentry.
```

> If you want to disable tracing for good you need to set <PlatformIdentifier name="traces-sample-rate" />`=None` or set <PlatformIdentifier name="enable-tracing" />`=False`. In this case now new traces will be started and no incoming traces will be continued.

Reviewer: ```suggestion
If you want to disable tracing for good you need to set <PlatformIdentifier name="traces-sample-rate" />`=None` or set <PlatformIdentifier name="enable-tracing" />`=False`. In this case no new traces will be started and no incoming traces will be continued.
```

Reviewer 2: Made a small tweak to @sentrivana 's suggestion.
